exercises/mongo-shell/basic-mongo-queries.md

```diff
@@ -514,33 +661,100 @@ db.movieDetails.find({"genres":["Action", "Adventure"]},
+* Spend **5 minutes** :alarm_clock: to find out what the movie `title` is in our collection! Feel free to use the internet to look up identifying information like the release year or the actor/director/writer names to use in your query filter. 
 
-For example, movies that were part of both the 'Action' and 'Adventure' genres! :boom: 
+* Once you have the name, let's update it to reflect the correct name "Taare Zameen Par". 
 
-Using commands like:
-- `show dbs`
-- `use <database>`
-- `show collections`
+Here's how to perform the update operation:
+```javascript
+db.movieDetails.updateOne(
+    {<your uniquely identifying filter as key: value pair>}, 
+    {"$set": {"title": "Taare Zameen Par"}}
+    )
+```
 
-Explore the different datasets available on your workstation.  Be creative and try to find interesting things.  
+**Exercise 16** :computer: 
 
-Some questions to consider while exploring:
-- How is this data structured?
-- Is there anything interesing to me about this data?
-- Is the data malformed, incorrect or incomplete?
-- What are the challenges you faced while exploring this data?
+MongoDB offers multiple operators that can be used in its update operations. In the exercise above, we looked at the `$set` operator. 
 
-**Presentation: :alarm_clock: 25 minutes**
-Each group will prepare a small, **5 minute**, 2-3 slide presentation about the datasets.
+|Name|Description|
+|---|---|
+|`$set`|Sets the value of a field in a document|
+|`$unset`|Removes the specified field from a document|
+|`$currentDate`|Sets the value of a field to current date, either as a Date or a Timestamp|
+|`$inc`|Increments the value of a field by a specified amount|
+|`$mul`|Multiplies the value of the field by the specified amount|
+|`$min`|Only updates the field if the specified value is less than the existing field value.|
+|`$max`|Only updates the field if the specified value is greater than the existing field value.|
+|`$setOnInsert`|Sets the value of a field if an update results in an insert of a document. Has no effect on update operations that modify existing documents.|
+|`$rename`|Renames a field|
+
+
+Take **10 minutes** :alarm_clock: to go through this list and write an interesting `$updateMany()` query. 
+
+
+:arrow_right: Note that `updateMany()` will make the same modification to all the documents that match the given filter. 
+
+---
+
+## Delete Operations
+
+The MongoDB query language supports 2 delete operations:
+* `deleteOne()`
+* `deleteMany()`
+
+**Exercise 17** :computer: 
 
-Present your results from **Lab 2** and answer any questions your classmates may have.
-Be sure to include your interesting movie :clapper: queries :+1:
+In our `movieDetails` collection, there is only 1 movie from the `year` 2018 with a `title` "QQ Speed". To perform the exercise, delete this record. 
```

Comment:
again, maybe we want this on a junk database.  Especially if any other exercise relies on counting or filtering on these parameters.  This is the **basic** exercise document, and I know you lean on the movies db heavily in the intermediate document.  

I also worry that you are stuck on this movies db because it's what you see in the videos.  Can we structure some activities around these labs to do the same things against other databases?

Reply:
Maybe we teach against the movie database and all the activities and group work revolve around the other databases they have?  In my head this is a much better option.

Reply:
Added suggestions to where `updateMany()` and `deleteMany()` operations are performed against the dummy `employee_info` collection that was created earlier in this document.